MovieViewController.swift

```diff
@@ -0,0 +1,295 @@
+//
+//  MovieViewController.swift
+//  movieViewer
+//
+//  Created by victoria_ku on 6/7/17.
+//  Copyright © 2017 victoria_ku. All rights reserved.
+//
+
+import UIKit
+import AFNetworking
+import MBProgressHUD
+
+class MovieViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+
+  @IBOutlet weak var tableView: UITableView!
+  var movies: [NSDictionary]?
+  var endpoint: NSString!
+  var pagenum = 1
+
+
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+  }
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+
+    override func viewDidLoad() {
+      super.viewDidLoad()
+
+      tableView.delegate = self
+      tableView.dataSource = self
+
+        // Do any additional setup after loading the view.
+
+      let refreshControl = UIRefreshControl()
+      refreshControl.addTarget(self, action: #selector(refreshControlAction(_:)), for: UIControlEvents.valueChanged)
+      tableView.insertSubview(refreshControl, at: 0)
+
+      let frame = CGRect(x: 0, y: tableView.contentSize.height, width: tableView.bounds.size.width, height: InfiniteScrollActivityView.defaultHeight)
+      loadingMoreView = InfiniteScrollActivityView(frame: frame)
+      loadingMoreView!.isHidden = true
+      tableView.addSubview(loadingMoreView!)
+
+      var insets = tableView.contentInset
+      insets.bottom += InfiniteScrollActivityView.defaultHeight
+      tableView.contentInset = insets
+
+
+      UINavigationBar.appearance().tintColor = UIColor.black
+
+      let apiKey = "a07e22bc18f5cb106bfe4cc1f83ad8ed"
+      let url = NSURL(string: "https://api.themoviedb.org/3/movie/\(endpoint!)?api_key=\(apiKey)&page=\(pagenum)")
```

Comment:
you usually don't want to force unwrap this.

it's nicer to optionally unwrap, that way you can also use the unwrapped variable on L58